1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -1,46 +1,46 @@
+ஒரு நிரலர்(programmer) தன்னுடைய அதிக நேரத்தை செலவிடும் இடம் குறிமுறை தொகுப்பி ஆகும்.
 
-There are two main types of code editors: IDEs and lightweight editors. Many people use one tool of each type.
+குறிமுறை தொகுப்பியில் இரண்டு முக்கிய வகைகள் உள்ளன: ஐடிஇ(IDE)க்கள் மற்றும் இலகுரக தொகுப்பிகள். ஒவ்வொரு வகையிலும் பலர் ஒரு கருவியைப் பயன்படுத்துகிறார்கள்.
 
-## IDE
+## ஐடிஇ(IDE)
 
-The term [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) (Integrated Development Environment) refers to a powerful editor with many features that usually operates on a "whole project." As the name suggests, it's not just an editor, but a full-scale "development environment."
+[ஐடிஇ] (https://en.wikipedia.org/wiki/Integrated_development_environment) (ஒருங்கிணைந்த மேம்பாட்டு சூழல்) என்பது "முழு திட்டப்பணியிலும்" பொதுவாக செயல்படும் பல பண்புகளைக் கொண்ட சக்திவாய்ந்த தொகுப்பியைக் குறிக்கிறது. அதன் பெயர் குறிப்பிடுவது போல, இது ஒரு தொகுப்பி மட்டுமல்ல, முழு அளவிலான "மேம்பாட்டு சூழல்".
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+ஐடிஇ ஒரு திட்டத்தை மின்னேற்றுகிறது (இது பல கோப்புகளை உள்ளடக்கியதாக இருக்கலாம்), கோப்புகளுக்கு இடையில் வழிசெலுத்தலை (navigation) அனுமதிக்கிறது, முழு திட்டத்தின் அடிப்படையில்  (திறந்த கோப்பு மட்டுமல்ல) தன்னியக்கத்தை வழங்குகிறது மற்றும் பதிப்புரு மேலாண்மை அமைப்பு([git](https://git-scm.com/) போன்றவை), சோதனை சூழல் மற்றும் பிற "திட்ட-நிலை" தேவைகளுடன் ஒருங்கிணைக்கிறது.
 
-If you haven't selected an IDE yet, consider the following options:
+நீங்கள் இன்னும் ஒரு ஐடிஇ ஐத் தேர்ந்தெடுக்கவில்லை என்றால், பின்வரும் விருப்பங்களை கவனியுங்கள்:
 
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [WebStorm](http://www.jetbrains.com/webstorm/) (cross-platform, paid).
+- [விஷுவல் ஸ்டுடியோ கோட்(Visual Studio Code)](https://code.visualstudio.com/) (குறுக்கு-தளம்(cross-platform), இலவச வகை).
+- [வெப்ஸ்டோர்ம்(WebStorm)](http://www.jetbrains.com/webstorm/) (குறுக்கு-தளம், கட்டண வகை).
 
-For Windows, there's also "Visual Studio", not to be confused with "Visual Studio Code". "Visual Studio" is a paid and mighty Windows-only editor, well-suited for the .NET platform. It's also good at JavaScript. There's also a free version [Visual Studio Community](https://www.visualstudio.com/vs/community/).
+விண்டோஸைப்(Windows) பொறுத்தவரை, "விஷுவல் ஸ்டுடியோ"(Visual Studio) என்பதும் உள்ளது, "விஷுவல் ஸ்டுடியோ கோட்" உடன் குழப்பமடையக்கூடாது. "விஷுவல் ஸ்டுடியோ" என்பது கட்டணம் செலுத்தவேண்டிய மற்றும் வலிமையான விண்டோஸ்-இல் மட்டுமே இயங்கக்கூடிய தொகுப்பி, இது .நெட்(.NET) இயங்குதளத்திற்கு மிகவும் பொருத்தமானது. இது ஜாவாஸ்கிரிப்ட்-டிற்கும் பொருத்தமானது. இதன் இலவச பதிப்பும் உள்ளது [விஷுவல் ஸ்டுடியோ சமூகம்(Visual Studio Community)](https://www.visualstudio.com/vs/community/).
```

Comment:
This seems more natural. Your take @DasarathanSampath ?

```suggestion
விண்டோஸைப்(Windows) பொறுத்தவரை, "விஷுவல் ஸ்டுடியோ"(Visual Studio) என்பதும் உள்ளது, அதை "விஷுவல் ஸ்டுடியோ கோட்" உடன் குழப்பிக்கொள்ள வேண்டாம். "விஷுவல் ஸ்டுடியோ" என்பது கட்டணம் செலுத்தவேண்டிய மற்றும் வலிமையான விண்டோஸ்-இல் மட்டுமே இயங்கக்கூடிய தொகுப்பி, இது .நெட்(.NET) இயங்குதளத்திற்கு மிகவும் பொருத்தமானது. இது ஜாவாஸ்கிரிப்ட்-டிற்கும் பொருத்தமானது. இதன் இலவச பதிப்பும் உள்ளது [விஷுவல் ஸ்டுடியோ சமூகம்(Visual Studio Community)](https://www.visualstudio.com/vs/community/).
```

Reply:
Instead of that can we say like this? @gokatz 
விண்டோஸைப்(Windows) பொறுத்தவரை, "விஷுவல் ஸ்டுடியோ"(Visual Studio) என்பதும் "விஷுவல் ஸ்டுடியோ கோட்" என்பதும் வேறு வேறானது.

Reply:
The above statement is straight forward. I like this, if you & Gokul are agreeing.